slurm/2_ApplicationSpecific/VASP-GPU-podman/build_vasp.bash

```diff
@@ -90,16 +90,22 @@ fi
 if [ -f "run_vasp_shell.bash" ]
 then
   sed -E -e "s|^vasp_version=.*|vasp_version=\"${vasp_version}\"|" \
-   -e "s|^image_tarball_dir=.*|image_tarball_dir=\"${tarball_dir}\"|" \
+   -e "s|^oci_archive_dir=.*|oci_archive_dir=\"${oci_archive_dir}\"|" \
    "run_vasp_shell.bash" > "${bin_dir}/run_vasp_shell-${vasp_version}.bash"
   chmod 755 "${bin_dir}/run_vasp_shell-${vasp_version}.bash"
   ln -sf "${bin_dir}/run_vasp_shell-${vasp_version}.bash" "${bin_dir}/run_vasp_shell"
 else
   echo "script \"run_vasp_shell.bash\" missing?" >&2
 fi
 # Provide a sample Slurm script
-if [ -f sample_vasp_std.bash ]
+if [ -f sample_SLURM_vasp_std.bash ]
 then
+  default_acct="$(sacctmgr -rnp show User "$(id -un)" | awk -F'|' '{print $2}')"
```

Comment:
Can you describe what this section and the next are intended for?  If the Slurm script exists then an account is included in it and a data path should be set in it.

Reply:
I think you are referring to this section:

```
if [ -f sample_SLURM_vasp_std.bash ]
then
  default_acct="$(sacctmgr -rnp show User "$(id -un)" | awk -F'|' '{print $2}')"
  sed -E -i -e "/#SBATCH[[:space:]]+--account=/s|\".*\"|\"${default_acct}\"|" \
   -e "/#SBATCH[[:space:]]+--chdir=/s|\".*\"|\"${base_dir}\"|" \
   -e "s|^bin_dir=.*|bin_dir=\"${bin_dir}\"|" \
   "sample_SLURM_vasp_std.bash"
  # try to provide a reasonable data dir path
  grp_group="$(groups | sed 's/ /\n/g' | grep ^grp- | head -1 | sed 's/^grp-//')"
  if [ "${grp_group}" != "" ]
  then
    if [ -d "/projects/academic/${grp_group}" ]
    then
      base_dir="/projects/academic/${grp_group}/$(id -un)/VASP"
      sed -E -i -e "s|^data_dir=.*|data_dir=\"${base_dir}/data\"|" \
       "sample_SLURM_vasp_std.bash"
    fi
  fi
  cp "sample_SLURM_vasp_std.bash" "${bin_dir}/sample_SLURM_vasp_std.bash"
else
  echo "sample script \"sample_SLURM_vasp_std.bash\" missing?" >&2
fi

```

This subsidies the users installed OCI directory in bin directory path into
the SLURM sample script, set the default SLURM account to the user's
group & sets sample paths based on the user's username & group